dataplane/linux/int_dataplane.go

```diff
@@ -268,6 +269,7 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		LockProbeInterval:     config.IptablesLockProbeInterval,
 		BackendMode:           backendMode,
 		LookPathOverride:      config.LookPathOverride,
+		OnStillAlive:          dp.reportHealth,
```

Comment:
Just a language point check: is `dp.reportHealth` valid as soon as `dp` has been constructed?  (I'm not sure why it wouldn't be, but just being paranoid.)

Reply:
Yes, I think that's fine in go.

Reply:
Thanks.